example-config.env

```diff
@@ -1,44 +0,0 @@
-PORT=3000
-
```

Comment:
why was this file deleted?

Reply:
Forgot to copy it and then rename it, my bad